packages/runtime/src/errors/errors-list.ts

```diff
@@ -237,6 +237,12 @@ by an index that does not exist.`
     message: "set bit index %index% is beyond bytes length. [error-line: %line%]",
     title: 'set bit index error',
     description: `set bit index error`
+  },
+  SCRATCH_EXIST_ERROR: {
+    number: 1035,
+    message: "scratch space doesn't exist for index: %index%. [error-line: %line%]",
+    title: 'scratch space error',
+    description: `fails maybe because the requested transaction is an ApplicationCall or T < GroupIndex.`
```

Comment:
message and description seems completely different here. Could you please add a reason in message as well?

Reply:
message says scratch space doesn't exist, description states that either that tx is not application call or tx is not even in group, how is it unrelated ?
I will add there a reason may be tag

Reply:
By different I meant the wording. During catch I think error message is shown, so I suggested to add a reason in message.

Reply:
ok